Description: Remove unused libipsec linking with libfl
 Due to the yylex requirement this caused FTBFS
 when a shared libfl is provided.
Author: Adrian Bunk <bunk@debian.org>

Index: pkg-ipsec-tools/src/libipsec/Makefile.am
===================================================================
--- pkg-ipsec-tools.orig/src/libipsec/Makefile.am
+++ pkg-ipsec-tools/src/libipsec/Makefile.am
@@ -26,7 +26,6 @@ libipsec_la_SOURCES = \
 # version is current:revision:age.
 # See: http://www.gnu.org/manual/libtool-1.4.2/html_chapter/libtool_6.html#SEC32
 libipsec_la_LDFLAGS = -version-info 0:1:0
-libipsec_la_LIBADD = $(LEXLIB)
 
 noinst_HEADERS = ipsec_strerror.h
 
